package com.sist.exception;

import javax.management.RuntimeErrorException;

public class MainClass3 {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		try {
			
		}
		catch(NumberFormatException ex) {
			
		}catch(ArrayIndexOutOfBoundsException ex) {
			
		}catch(ArithmeticException ex) {
			
		}catch(RuntimeErrorException ex) {
			
		}
	}

}
